transformer_engine/common/fused_softmax/scaled_upper_triang_masked_softmax.cu

```diff
@@ -249,7 +249,7 @@ template <typename input_t, typename output_t, typename acc_t, int log2_elements
 __global__ void scaled_upper_triang_masked_softmax_warp_backward(
     output_t *gradInput,
     input_t *grad,
-    const input_t *output,
+    input_t *output,
```

Comment:
I don't believe it is correct - looking at the kernel "output" is actually an input (since it seems to be the softmax output, not the output of the backward of softmax), while the actual output is "gradInput". This is also consistent with the types (`input_t` for `output` and `output_t` for `gradInput`). So that const should be here.